README.md

```diff
@@ -39,13 +38,10 @@ The core team operate commercially through http://www.dwyl.io/ - an IT Services
 [_**Join us**_](#how) in finding a problem you want to **solve** and will
 _**love**_ working on.
 
-
 ## What?
 
 **Q**: What is dwyl?  
-**A**: dwyl (pronounced "[Will](https://www.youtube.com/watch?v=0sxnonO3FXw)"
-but with a "D" infront of it) is a _community_ of people on a
-_**mission to change the world using technology**_.
+**A**: dwyl (pronounced "[Will](https://www.youtube.com/watch?v=0sxnonO3FXw)" but with a "D" in front of it) is a _community_ of people on a _**mission to change the world using technology**_.
```

Comment:
The reason we prefer splitting long sentences up into multiple lines is https://sembr.org 💭